_overviews/collections-2.13/arrays.md

```diff
@@ -8,7 +8,7 @@ num: 10
 previous-page: concrete-mutable-collection-classes
 next-page: strings
 
-languages: [ru]
+languages: [uk, ru]
```

Comment:
Why  `uk` and not `ukr`?

Reply:
There's only ISO 639-1 code used, so we are also using it.